src/app/page.module.css

```diff
@@ -9,3 +9,15 @@
   margin-top: 4px;
   margin-bottom: 4px;
 }
+
+.CarouselImgContainer {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin: 20px;
+}
+@media (max-width: 390px) {
```

Comment:
Use breakpoints in accordance with Fluent 2 guidelines. Ref: https://fluent2.microsoft.design/layout#responsive-design